actions/DisplayAction.php

```diff
@@ -153,7 +153,7 @@ public function execute() {
 					'donationUri'  => $bridge->getDonationURI(),
 					'icon' => $bridge->getIcon()
 				);
-			} catch(Error $e) {
+			} catch(\Throwable $e) {
```

Comment:
2. The special casing of 0 code (i.e. “Error 0 is a special case”).

Since it is a [default error code for `Exception`](https://www.php.net/manual/en/exception.construct.php), this will likely lead to wrong error message (“unexpected situation” rather than “error”).

I suggest

```diff
-						if($e->getCode() === 0) {
+						if($e instanceof \Error && $e->getCode() === 0) {
```

Reply:
Don't worry about this. Will remove later.Not much value to differentiate on the error codes. The current behavior just gives a slight difference in the error message.

Reply:
Then why not remove it now?

In my experience, commits should either contain refactoring producing operationally equivalent code, or changing the functionality. Otherwise it is just a mess for reviewers and people in the future trying to find why the change was made. This changeset appears on the surface to do the former, while doing badly the latter.

Reply:
I think my PR is actually a 100% refactor because the removed catch statement is never invoked. Since `Error` implements `Throwable` the first catch statement is catching everything.

Reply:
As I mentioned, your change switches the error title to a more confusing one for majority of `Exception`s (those that do not set error code).

Reply:
Okay I see the difference now. Added a commit.